main.go

```diff
@@ -388,6 +395,10 @@ type server struct {
 	ServerFeatures []string `json:"server_features,omitempty"`
 }
 
+// Authority
```

Comment:
Please add a complete comment.

Reply:
it didnt let me do the TODO without the `authority` in there. i'll add something in